tests/test_pipedrive_pagination.py

```diff
@@ -9,12 +9,12 @@ def name():
         return "tap_tester_pipedrive_pagination_test"
 
     def test_run(self):
-        # page size for "deals"
+        # page size for "pipelines"
```

Comment:
We do not have enough records for `deals` stream to test the pagination. So, replaced it with `pipeline` stream.